wordpress-mobile/AztecEditor-iOS
# Improve “More” button accessibility

Refs https://github.com/wordpress-mobile/WordPress-iOS/issues/12953

Currently, when using VoiceOver, hovering on the _More_ button would make VoiceOver speak “Possibly, More“:

<img src="https://user-images.githubusercontent.com/198826/70292277-f9b4ce80-179a-11ea-922d-3c89bde67d56.png" width="320">

This changes that so that it can be configured to say “More”. 

The button will now also report to VoiceOver that the `FormatBar` is currently expanded or not. When hovering on an expanded bar's More button, VoiceOver would speak “selected“. 

<img src="https://user-images.githubusercontent.com/198826/70292359-48faff00-179b-11ea-82eb-59ce555b19bd.png" width="320">

## Testing

1. Navigate to the Standard Demo.
2. Turn VoiceOver on. Please confirm the changes described above. 

## Reviewing 

Based on what I understand, the localized “More” accessibility label should be configured in the hosting app (e.g. WPiOS). In here, I added a configurable `FormatBar` property called `overflowToggleAccessibilityLabel`. Please let me know if there's a more appropriate way to do this.

## Submitter Checklist
